src/svc-hooks/create-init.ts

```diff
@@ -0,0 +1,79 @@
+import * as fs from "fs";
+import { join } from "path";
+
+const YAML = require("json2yaml");
+
+export const command = "create-init";
+export const desc = "Initialize the YAML file for creating service hooks";
+export const builder = (yargs: import("yargs").Argv) =>
+  yargs
+    .option("outDir", {
+      alias: "o",
+      default: ".",
+      describe: "Directory to write output file to",
+      type: "string",
+      normalize: true,
+    })
+    .option("outType", {
+      alias: "t",
+      default: "yaml",
+      describe: "Output file type",
+      type: "string",
+      choices: ["yaml", "json"],
+    });
+export function handler(argv: any) {
+  // Some dummy data to output for user to fill in.
+  const sampleJSON = [
+    {
+      org: null,
+      project: null,
+      eventType: "build.complete",
+      url: null,
+      eventSpecificArgs: {
+        definitionName: null,
+        buildStatus: null,
+        publisherId: "tfs",
+      },
+    },
+    {
+      org: null,
+      project: null,
+      eventType: "ms.vss-pipelines.stage-state-changed-event",
+      url: null,
+      eventSpecificArgs: {
+        pipelineId: null,
+        stageNameId: null,
+        stageStateId: null,
+        publisherId: "pipelines",
+      },
+    },
+    {
+      org: null,
+      project: null,
+      eventType: "git.push",
+      url: null,
+      eventSpecificArgs: {
+        repoName: null,
+        branch: null,
+        pushedBy: null,
+        publisherId: "tfs",
+      },
+    },
+  ];
+
+  // Write sample to output file
+  const fPath = join(
+    argv.outDir,
+    `create.${argv.outType === "yaml" ? "yml" : "json"}`
```

Comment:
No, I always though YML was the official one since it was 3 letters, just like EXE, TXT, etc. I mean it's just a file extension anyway, but I can change it back. It'll be way easier to read anyway.

Reply:
Uses .yaml now. Doesn't use .yml at all.